pkg/storage/export/virt-exportserver/exportserver.go

```diff
@@ -475,6 +475,11 @@ func archiveHandler(mountPoint string) http.Handler {
 			w.WriteHeader(http.StatusBadRequest)
 			return
 		}
+		if hasPermissions := checkDirectoryPermissions(mountPoint); !hasPermissions {
+			w.WriteHeader(http.StatusForbidden)
```

Comment:
I think 5xx error codes are more appropriate in this case.  4xx errors indicate an issue with the client.  See:  https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_client_errors